source/scripting_v3/GTA/Camera/Camera.cs

```diff
@@ -555,6 +555,19 @@ public void AttachTo(PedBone pedBone, Vector3 offset)
 			Function.Call(Hash.ATTACH_CAM_TO_PED_BONE, Handle, pedBone.Owner.Handle, pedBone, offset.X, offset.Y, offset.Z, true);
 		}
 
+		/// <summary>
+		/// Attaches this <see cref="Camera"/> to a specific bone on a specific <see cref="Vehicle"/>.
+		/// </summary>
+		/// <param name="vehicleBone">The <see cref="Vehicle"/> bone to attach the camera to.</param>
+		/// <param name="offset">Offset relative to the specified <paramref name="bone"/>.</param>
+		/// <param name="rotation">Rotation vector of the <see cref="Camera"/>.</param>
+		/// <param name="relativeRotation">Set <see langword="true" /> to have rotation relative to the specified <paramref name="bone"/>; else set <see langword="false" />.</param>
+		/// <param name="fixedDirection">Set <see langword="true" /> to have camera direction match the <see cref="Vehicle"/> direction; else set <see langword="false" />.</param>
+		public void AttachTo(EntityBone vehicleBone, Vector3 offset, Vector3 rotation, bool relativeRotation, bool fixedDirection)
```

Comment:
I noticed there are other camera attach natives that can be applied to vehicle bones, which is `ATTACH_CAM_TO_PED_BONE` and `HARD_ATTACH_CAM_TO_PED_BONE`. IMO, having separate name like `AttachToVehicleBone` would be better so we won't have trouble in overload resolution.

As for bool param names, we should rename the 1st bool argument (the 3rd one of all params of the native) to like `hardAttachment`. That param determines if the cam will be hard attached so rotation values of the bone will also be used. Although this param is set to true in all occurrences of all ysc scripts for b2944, we can't workaround without letting people specify this param. The 2nd one (the last one of all params of the native) is officially named `bOffsetIsRelative`, but it looks like R* sets this kind of param to true almost all the time if any. Therefore, maybe we could just omit the bool param for relative offset and set to true implicitly (this one is also set to true in all occurrences of all ysc scripts for b2944). You can convert local space to world space if really needed.

Again, this is the official doc of `ATTACH_CAM_TO_VEHICLE_BONE`:
```
//INFO: 
//PARAM NOTES: 
//  vehicleIndex: The index of the vehicle to attach the camera to.
//	vehicleBoneIndex: The index of the vehicle bone to attach to.
//	hardAttachment: if set to true, will fully attach the camera to the bone transform, otherwise only the position of the bone will be retained.
//	vecRotationOffset: An additional rotational offset to be applied from the attach bone rotation (x=yaw, y=pitch, z=roll). This is only considered if hardAttachment is set to TRUE.
//	vecPositionOffset: An additional offset to be applied from the attach position.
//	bOffsetIsRelative: If true, the offsets are applied relative to the transform of the attached vehicle (not the bone), rather than in world-space.
//PURPOSE: Attaches a camera to a vehicle's bone.
NATIVE PROC ATTACH_CAM_TO_VEHICLE_BONE(CAMERA_INDEX CameraIndex, ENTITY_INDEX vehicleIndex, INT vehicleBoneIndex, BOOL hardAttachment, VECTOR vecRotationOffset, VECTOR vecPositionOffset, BOOL bOffsetIsRelative) = "0xbe3ecd06b6bdf192"
```